nncf/common/graph/transformations/commands.py

```diff
@@ -140,6 +140,7 @@ def __init__(self, target_type: TargetType):
         :param target_type: Type of the target point.
         """
         self._target_type = target_type
+        self.target_node_name: str = ""
```

Comment:
I suggest to add the `target_node_name` to the initialization parameters list as all inherited classes requires this parameter. Don't forget to update the docstring